README.md

```diff
@@ -229,13 +245,17 @@ To run a specific set of tests:
 ```shell
 yarn test:node
 yarn test:browser
+yarn test:integration
```

Comment:
might be worth mention when running nested `test:*` targets, those require needing to manually run `build` prior to compile the latest local code version otherwise the test will run against whatever was last compiled, whereas `test` does rebuild source automatically.

Reply:
great callout! fixed